go.mod

```diff
@@ -21,19 +21,19 @@ require (
 	google.golang.org/genproto v0.0.0-20220725144611-272f38e5d71b // indirect
 )
 
+require cosmossdk.io/math v1.0.0-beta.3
```

Comment:
This should be included in the require list above.

Reply:
huh, weird not sure why goland decided to put that there 🤔 fixed here https://github.com/regen-network/regen-ledger/pull/1357/commits/8e4a094ef8d6b33ac79fc2fa3bfd687a1379083f